docs/getting_the_most_out_of_securedrop.rst

```diff
@@ -91,7 +91,7 @@ proof of concept Twitter advertisement aimed at EPA and NOAA employees to show
 how it can be done. You can read about `how you can do the same thing here`_.
 
 .. _`recently targeted online advertisements`: https://www.wsj.com/articles/gizmodo-ads-target-potential-trump-leakers-1487191482
-.. _`tell on trump`: https://web.archive.org/web/20201114212453/https://specialprojectsdesk.com/tell-on-trump-1792401813
+.. _`tell on trump`: https://web.archive.org/web/20201114214553/https://specialprojectsdesk.com/tell-on-trump-1792401813
```

Comment:
This returns a 200 from the IA, but the recorded page is a Kinja 404. Using an earlier version like https://web.archive.org/web/20200926063152/https://specialprojectsdesk.com/tell-on-trump-1792401813 works.

Reply:
I picked the last date that didn't display the Kinja 404 page, so the green/corrected link shouldn't  do that (Nov. 14) - though the [misspelled URL sometimes did](https://github.com/freedomofpress/securedrop-docs/issues/126). **That being said**, the September 26 version is just as fine. I'll make the change :+1: